.github/workflows/regression-tests.yml

```diff
@@ -141,16 +141,15 @@ jobs:
         sudo make install
     - name: Set up Icarus (Windows - source)
       if: startsWith(matrix.os, 'windows') && matrix.sim == 'icarus'
-      timeout-minutes: 5
+      timeout-minutes: 7
       run: |
         conda install -c msys2 m2-base m2-make m2-autoconf m2-flex m2-bison m2-gperf m2w64-toolchain
         git clone https://github.com/steveicarus/iverilog.git
         cd iverilog
         git reset --hard ${{matrix.sim-version}}
         bash ./autoconf.sh
         bash ./configure --host=x86_64-w64-mingw32 --prefix=/c/iverilog
-        make -j $(nproc)
-        make install
+        make -j $(nproc) install
```

Comment:
IDK about this. It looks like Icarus is being built twice? I'm wondering if this will help prevent that.

Reply:
Yeah something here seems wonky, but I don't have time to diagnose it now.